lua/wikis/marvelrivals/Infobox/Person/Player/Custom.lua

```diff
@@ -0,0 +1,180 @@
+end
+
+---@param id string
+---@param widgets Widget[]
+---@return Widget[]
+function CustomInjector:parse(id, widgets)
+	local caller = self.caller
+	local args = caller.args
+
+	if id == 'custom' then
+		local heroes = Array.sub(caller:getAllArgsForBase(args, 'hero'), 1, MAX_NUMBER_OF_SIGNATURE_HEROES)
+		local heroIcons = Array.map(heroes, function(hero)
+			return CharacterIcon.Icon{character = CharacterNames[hero:lower()], size = SIZE_HERO}
+		end)
+
+		Array.appendWith(widgets,
+			Cell{
+				name = #heroIcons > 1 and 'Signature Heroes' or 'Signature Hero',
+				content = {table.concat(heroIcons, '&nbsp;')},
+			}
+		)
+	elseif id == 'role' then
+		return {
+			Cell{name = 'Role', content = {
+				caller:_displayRole(caller.role),
+				caller:_displayRole(caller.role2),
+			}},
+		}
+	end
+
+	return widgets
+end
+
+---@param role string?
+---@return {category: string, variable: string, isplayer: boolean?}?
+function CustomPlayer:_getRoleData(role)
+	return ROLES[(role or ''):lower()]
+end
+
+---@param roleData {category: string, variable: string, isplayer: boolean?}?
+---@return string?
+function CustomPlayer:_displayRole(roleData)
+	if not roleData then return end
+
+	return Link{
+		link = ':Category:' .. roleData.category,
+		children = roleData.variable
+	}
+end
+
+---@param args table
+function CustomPlayer:defineCustomPageVariables(args)
+	Variables.varDefine('role', (self.role or {}).variable)
+	Variables.varDefine('role2', (self.role2 or {}).variable)
+end
+
+---@param categories string[]
+---@return string[]
+function CustomPlayer:getWikiCategories(categories)
+	return Array.append(categories,
+		(self.role or {}).category,
+		(self.role2 or {}).category
+	)
+end
+
+---@param lpdbData table
+---@param args table
+---@param personType string
+---@return table
+function CustomPlayer:adjustLPDB(lpdbData, args, personType)
+	lpdbData.extradata.role = (self.role or {}).variable
+	lpdbData.extradata.role2 = (self.role2 or {}).variable
+
+	-- store signature heroes with standardized name
+	Table.mergeInto(lpdbData.extradata, Table.map(
+		Array.sub(self:getAllArgsForBase(args, 'hero'), 1, MAX_NUMBER_OF_SIGNATURE_HEROES),
+		function(index, hero) return 'signatureHero' .. index, CharacterNames[hero:lower()]
+	end))
+
+	lpdbData.type = self:_isPlayerOrStaff()
+
+	lpdbData.region = Region.name({region = args.region, country = args.country})
+
+	if String.isNotEmpty(args.team2) then
+		lpdbData.extradata.team2 = args.team2
+	end
+
+	return lpdbData
+end
+
+---@return string
+function CustomPlayer:_isPlayerOrStaff()
+	local roleData
+	if String.isNotEmpty(self.args.role) then
+		roleData = ROLES[self.args.role:lower()]
+	end
+	-- If the role is missing, assume it is a player
+	if roleData and roleData.isplayer == false then
+		return 'staff'
+	else
+		return 'player'
+	end
+end
+
+---@return string?
+function CustomPlayer:createBottomContent()
+	if self:shouldStoreData(self.args) and String.isNotEmpty(self.args.team) then

```

Comment:
invert the condition and return early (empty)
that way you can decrease indent by 1